### DISCUSSION TITLE
Request to change PurgeCSS Option from Whitelist to Allowedlist (or similar)

### DISCUSSION THREAD

**Discussion by jodiedoubleday** [tailwindlabs/tailwindcss#1969] [General]
It goes without saying that the terminology whitelist and blacklist has no place in society anymore and it's based on unethical and outdated ideas that white is allowed and black is not.

In line with this thinking, I'd love for Tailwind to move to a more suitable blocked/allowed type naming structure.

✊🏻✊🏽✊🏾✊🏿

**Comment by adamwathan** [comment]
See existing issue:

https://github.com/tailwindcss/tailwindcss/issues/1868

We don't use this terminology in the Tailwind codebase anywhere (all we have is an "options" key for PurgeCSS that transparently forwards options to PurgeCSS, so people can configure it directly.) Waiting on resolution on PurgeCSS's end before we add any sort of layer on top to make sure we don't end up with divergent APIs.
